ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml

```diff
@@ -44,16 +44,14 @@ tests:
     dependency_overrides:
       OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.14
     env:
-      CREDENTIALS_MODE: Passthrough
-      FEATURE_SET: TechPreviewNoUpgrade
-      SLEEP_DURATION: 8h
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      SLEEP_DURATION: 2h
     post:
     - ref: cucushift-installer-wait
-    - ref: storage-destroy-csi-gcp-filestore
-    - chain: cucushift-installer-rehearse-gcp-ipi-deprovision
+    - chain: cucushift-installer-rehearse-aws-ipi-proxy-deprovision
     pre:
-    - chain: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision
-    - chain: storage-conf-csi-optional-gcp-filestore
+    - chain: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision
+    - ref: fips-check-node-scan
```

Comment:
Any reason to modify other lines besides fips related stuff?
And `cluster_profile: gcp-qe` is there but you change to aws?

Reply:
No, actually just want to find a cluster to test without trigger all e2e test. Let me update it back.